mslib/mswms/mpl_hsec.py

```diff
@@ -330,17 +331,18 @@ def plot_hsection(self, data, lats, lons, bbox=(-180, -90, 180, 90),
                     del BASEMAP_CACHE[key]
                     BASEMAP_REQUESTS[:] = [_x for _x in BASEMAP_REQUESTS if key != _x]
 
-        # Set up the map appearance.
-        bm.drawcoastlines(color='0.25')
-        bm.drawcountries(color='0.5')
-        bm.drawmapboundary(fill_color='white')
-
-        # zorder = 0 is necessary to paint over the filled continents with
-        # scatter() for drawing the flight tracks and trajectories.
-        # Curiously, plot() works fine without this setting, but scatter()
-        # doesn't.
-        bm.fillcontinents(color='0.98', lake_color='white', zorder=0)
-        self._draw_auto_graticule(bm)
+        if self._plot_countries:
+            # Set up the map appearance.
+            bm.drawcoastlines(color='0.25')
+            bm.drawcountries(color='0.5')
+            bm.drawmapboundary(fill_color='white')
```

Comment:
'white' is now a constant, does this mean that the option from the Map Appeareance is disabled?

Reply:
It was a constant before as well. No change here, I only made this code section optional in derived classes.

I think you confuse the client and server. The options affect only the client, this is server code.